### DISCUSSION TITLE
Date (+ optional time)

### DISCUSSION THREAD

**Discussion by say2joe** [community/community#126977] [Mobile] [answered]
Why isn't there a visible (last commit) date next to the files when browsing in the "code" section via the mobile app. It's infuriating not knowing the date of any files while browsing in the mobile app.

**Comment by esahmed21** [comment, marked as answer]
Hi @say2joe 👋

That's a great call. I'll create an internal issue for our team to track it and consider it. This would certainly enhance the user experience by providing more context while browsing files.

Thank you for your valuable feedback!

**Reply by say2joe** [reply]
In addition, the settings/preferences for the GitHub Mobile app are not considerable or overwhelming. If there's only minimal tech debt/loe involved (to add the commit date of files to the UI, I'm erring on the assumption it is), perhaps this and other future file properties could be optionally displayed if the user selected an option in settings - thus minimizing QA testing and usability nuances for a quicker release (and less scope creep to your backlog or prioritized user stories) - I strongly believe that the commit date is essential, but, nonetheless, it's an opinion derived in no small way from the fact that I've got a 6.8" smartphone which includes a stylus (allowing me to get to those UI "nooks & crannies"). Regardless of the importance of that file's properties, it may simply be prohibitive for some users to have that much info displayed when browsing file repos. But, don't count out the other half of us (including those using tablets or TVs). Thanks for your reply regardless of your ability to facilitate this change request. It's nice to be acknowledged even if something may wind up being unattainable.
